### PR TITLE
chore: update dev-cn dockerfile

### PR DESCRIPTION
**Pull request by Benjamin-eecs**
## Description

- set PyPI mirror site in China
- set apt source list mirror site in China

## Motivation and Context

To make it faster when install dependencies

- [ ] I have raised an issue to propose this change ([required](https://envpool.readthedocs.io/en/latest/pages/contributing.html) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Chores

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://envpool.readthedocs.io/en/latest/pages/contributing.html) guide (**required**)
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the code using `make lint` (**required**)
- [x] I have ensured `make bazel-test` pass. (**required**)

### CODE REVIEW

File: envpool/classic_control/pendulum.h [hunk 1]
```diff
@@ -77,19 +77,18 @@ class PendulumEnv : public Env<PendulumEnvSpec> {
   void Step(const Action& action) override {
     done_ = (++elapsed_step_ >= max_episode_steps_);
     float act = action["action"_];
-    double u = act < -kMaxTorque  ? -kMaxTorque
-               : act > kMaxTorque ? kMaxTorque
-                                  : act;
+    double u =
+        act < -kMaxTorque ? -kMaxTorque : act > kMaxTorque ? kMaxTorque : act;
```
Comment (Benjamin-eecs) [inline envpool/classic_control/pendulum.h:81]: that is what 'make format' did

File: envpool/classic_control/pendulum.h [hunk 2]
```diff
@@ -77,19 +77,18 @@ class PendulumEnv : public Env<PendulumEnvSpec> {
...
     double new_theta_dot =
         theta_dot_ + 3 * (kGravity / 2 * std::sin(theta_) + u) * kDt;
     if (version_ == 0) {
       theta_ += new_theta_dot * kDt;
     }
-    theta_dot_ = new_theta_dot < -kMaxSpeed  ? -kMaxSpeed
-                 : new_theta_dot > kMaxSpeed ? kMaxSpeed
-                                             : new_theta_dot;
+    theta_dot_ = new_theta_dot < -kMaxSpeed
+                     ? -kMaxSpeed
```
Comment (Benjamin-eecs) [inline envpool/classic_control/pendulum.h:90]: here too

File: envpool/core/env_spec.h [hunk 3]
```diff
@@ -51,8 +51,8 @@ class EnvSpec {
   using Config = decltype(ConcatDict(common_config, EnvFns::DefaultConfig()));
   using ConfigKeys = typename Config::Keys;
   using ConfigValues = typename Config::Values;
-  using StateSpec = decltype(ConcatDict(
-      common_state_spec, EnvFns::StateSpec(std::declval<Config>())));
+  using StateSpec = decltype(
+      ConcatDict(common_state_spec, EnvFns::StateSpec(std::declval<Config>())));
```
Comment (Benjamin-eecs) [inline envpool/core/env_spec.h:55]: same here

File: envpool/classic_control/pendulum.h (hunk 1 above)
Comment (Benjamin-eecs) [reply envpool/classic_control/pendulum.h:81]: maybe there is some conflict between 'make format' and 'make lint'

Comment (Trinkle23897) [reply envpool/classic_control/pendulum.h:81]: I think it's because of version mismatch?

**Review by Trinkle23897 — APPROVED** [review]

**Pull request merged**